plugins/rest/komand_rest/actions/patch/action.py

```diff
@@ -15,10 +16,27 @@ def __init__(self):
         )
 
     def run(self, params={}):
+        """
+        If both inputs exist throw pluginException
+        Otherwise determine which one is empty
+        Send non-empty data
+        """
+        body_non_array = params.get(Input.BODY, {})
+        body_array = params.get(Input.BODY_AS_AN_ARRAY, [])
+        if not body_array:
+            data = body_non_array
+        elif not body_non_array:
+            data = body_array
+        elif body_array and body_non_array:
+            raise PluginException(
+                cause="You cannot send both inputs",
+                assistance="Try sending data either as an array OR an object, not both",
+            )
```

Comment:
Let's reverse condition, in my opinion it's more readable 
```suggestion
        if body_array and body_non_array:
            raise PluginException(
                cause="You cannot send both inputs",
                assistance="Try sending data either as an array OR an object, not both",
            )
        elif body_array:
            data = body_array
        elif body_non_array:
            data = body_non_array
```
And think about else condition when user will not write both (body_array, body_non_array) if that needed. 
I see it's needed because you set data in :39 line

Reply:
I agree that this is cleaner structure. I like to first check for failure cases and throw exceptions if found, and then let the expected/normal case run as normal (that is, not in an `else:`, but simply after the failure cases).